craft_parts/executor/organize.py

```diff
@@ -60,8 +60,6 @@ def organize_files(
 
             if os.path.isdir(src) and "*" not in key:
                 file_utils.link_or_copy_tree(src, dst)
-                # TODO create alternate organization location to avoid
```

Comment:
I was confused by this TODO just this week!

Reply:
Frankly I don't know what this means.